src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -53,51 +53,47 @@ Getting access to the image file is simple.
 Once you have access to the file you can do anything you want with it. For 
 example, you can:
 
-* Attach it directly to an `<canvas>` element so that you can manipulate it
+* Attach it directly to a `<canvas>` element so that you can manipulate it
 * Download it to the user's device
 * Upload it to a server by attaching to an `XMLHttpRequest` 
 
-Whilst using the input element method of getting access to image is 
-ubiquitous, it is the least appealing option because on desktop at least
-it doesn't access the user's Web Cam and it is not integrated directly into 
-your web page.
+Whilst using the input element method of getting access to images is 
+ubiquitous, it is the least appealing option because it is not integrated 
+directly into the web page, and on desktop can't access the user's webcam.
 
 ## Access the camera interactively
 
-Modern browsers can have a direct line to the camera allowing us to build
-experiences that are fully integrated with the web page and the user will never
+Modern browsers can get direct acccess to cameras, allowing us to build
+experiences that are fully integrated with the web page, so the user need never
 leave the browser.
 
-Warning: Direct access to the camera is a powerful feature, it requires consent from the user and your site needs to be on a secure origin (https)
+Warning: Direct access to the camera is a powerful feature, it requires consent 
+from the user and your site needs to be on a secure origin (HTTPS).
 
 ### Acquire access to the camera
 
-We can directly access the Microphone by using an API in the WebRTC 
-specification called `getUserMedia()`. `getUserMedia()` will prompt the user for 
+We can directly access a camera and microphone by using an API in the WebRTC 
+specification called `getUserMedia()`. This will prompt the user for 
 access to their connected microphones and cameras.
 
-If successful the API will return a `Stream` that will contain the data from
-the camera, and we can then either attach it to an `<video>` element and play it
-to show a real time preview or we could attach it to a `<canvas>` to get a
+If successful the API will return a `MediaStream` that contains data from
+the camera, and we can then either attach it to a `<video>` element and play it
+to show a real time preview, or attach it to a `<canvas>` to get a
 snapshot.
 
```

Comment:
the promise needs a catch. Maybe show how to check if a camera is present as well or at least hint to it and point to https://webrtc.github.io/samples/src/content/devices/input-output/ ?